### ISSUE TITLE
[Bug]: Installing MVTec - ValueError: Access denied: Path is outside the allowed directory

### ISSUE THREAD

**Issue by Issa-Bqain** [openvinotoolkit/anomalib#1804]
### Describe the bug

I am running the getting started notebook in my C drive. On running the dataset download section of the notebook, I am unable to install the dataset. I understand this may be due to some security check but this does not really resolve this issue - surely new users of this library should be able to run this notebook on their C drive??

**I run:** 

```
datamodule = MVTec()
#datamodule = MVTec(root="C:\Windows\MVTEC")
datamodule.prepare_data()  # Downloads the dataset if it's not in the specified `root` directory
datamodule.setup()  # Create train/val/test/prediction sets.

i, data = next(enumerate(datamodule.val_dataloader()))
print(data.keys())
```

**I get:**

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
[c:\code\anomalib_test\anomalib\notebooks\000_getting_started\001_getting_started.ipynb](file:///C:/code/anomalib_test/anomalib/notebooks/000_getting_started/001_getting_started.ipynb) Cell 13 line 3
      [1](vscode-notebook-cell:/c%3A/code/anomalib_test/anomalib/notebooks/000_getting_started/001_getting_started.ipynb#X16sZmlsZQ%3D%3D?line=0) datamodule = MVTec(root="C:\Windows\MVTEC")
      [2](vscode-notebook-cell:/c%3A/code/anomalib_test/anomalib/notebooks/000_getting_started/001_getting_started.ipynb#X16sZmlsZQ%3D%3D?line=1) datamodule.prepare_data()  # Downloads the dataset if it's not in the specified `root` directory
----> [3](vscode-notebook-cell:/c%3A/code/anomalib_test/anomalib/notebooks/000_getting_started/001_getting_started.ipynb#X16sZmlsZQ%3D%3D?line=2) datamodule.setup()  # Create train/val/test/prediction sets.
      [5](vscode-notebook-cell:/c%3A/code/anomalib_test/anomalib/notebooks/000_getting_started/001_getting_started.ipynb#X16sZmlsZQ%3D%3D?line=4) i, data = next(enumerate(datamodule.val_dataloader()))
      [6](vscode-notebook-cell:/c%3A/code/anomalib_test/anomalib/notebooks/000_getting_started/001_getting_started.ipynb#X16sZmlsZQ%3D%3D?line=5) print(data.keys())

File [C:\code\anomalib_test\anomalib\src\anomalib\data\base\datamodule.py:103](file:///C:/code/anomalib_test/anomalib/src/anomalib/data/base/datamodule.py:103), in AnomalibDataModule.setup(self, stage)
     96 """Set up train, validation and test data.
     97 
     98 Args:
     99     stage: str | None:  Train/Val/Test stages.
    100         Defaults to ``None``.
    101 """
    102 if not self.is_setup:
--> 103     self._setup(stage)
    104 assert self.is_setup

File [C:\code\anomalib_test\anomalib\src\anomalib\data\base\datamodule.py:121](file:///C:/code/anomalib_test/anomalib/src/anomalib/data/base/datamodule.py:121), in AnomalibDataModule._setup(self, _stage)
    118 assert self.train_data is not None
    119 assert self.test_data is not None
--> 121 self.train_data.setup()
    122 self.test_data.setup()
    124 self._create_test_split()

File [C:\code\anomalib_test\anomalib\src\anomalib\data\base\dataset.py:168](file:///C:/code/anomalib_test/anomalib/src/anomalib/data/base/dataset.py:168), in AnomalibDataset.setup(self)
    166 """Load data/metadata into memory."""
    167 if not self.is_setup:
--> 168     self._setup()
    169 assert self.is_setup, "setup() should set self._samples"

File [C:\code\anomalib_test\anomalib\src\anomalib\data\image\mvtec.py:236](file:///C:/code/anomalib_test/anomalib/src/anomalib/data/image/mvtec.py:236), in MVTecDataset._setup(self)
    235 def _setup(self) -> None:
--> 236     self.samples = make_mvtec_dataset(self.root_category, split=self.split, extensions=IMG_EXTENSIONS)

File [C:\code\anomalib_test\anomalib\src\anomalib\data\image\mvtec.py:131](file:///C:/code/anomalib_test/anomalib/src/anomalib/data/image/mvtec.py:131), in make_mvtec_dataset(root, split, extensions)
    128 if extensions is None:
    129     extensions = IMG_EXTENSIONS
--> 131 root = validate_path(root)
    132 samples_list = [(str(root),) + f.parts[-3:] for f in root.glob(r"**/*") if f.suffix in extensions]
    133 if not samples_list:

File [C:\code\anomalib_test\anomalib\src\anomalib\data\utils\path.py:217](file:///C:/code/anomalib_test/anomalib/src/anomalib/data/utils/path.py:217), in validate_path(path, base_dir, should_exist)
    215 if not str(path).startswith(str(base_dir)):
    216     msg = "Access denied: Path is outside the allowed directory"
--> 217     raise ValueError(msg)
    219 # In case path ``should_exist``, the path is valid, and should be
    220 # checked for read and execute permissions.
    221 if should_exist:
    222     # Check if the path exists

ValueError: Access denied: Path is outside the allowed directory
```

### Dataset

MVTec

### Model

N/A

### Steps to reproduce the behavior

run the getting started notebook using anomalib v1.0.0.dev0

### OS information

OS information:
- OS: Windows 10
- Anomalib version: 1.0.0.dev0

### Expected behavior

I expect it to download the dataset

### Pip/GitHub

GitHub

### Configuration YAML

```yaml
n/a
```

### Logs

```shell
check above
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samet-akcay** [comment]
We have merged a PR. Can you pulled the latest changes?

https://github.com/openvinotoolkit/anomalib/pull/1797

**Comment by Issa-Bqain** [comment]
> We have merged a PR #1797. Can you pulled the latest changes?

Just ran with the new release, works, thank you.

**Comment by Issa-Bqain** [comment]
Issue has been fixed in latest 1.0.0.dev0 release
